deployments/contractNames.ts

```diff
@@ -0,0 +1,42 @@
+export const CONTRACT = {
+  BitEthInitialCollateralSet: 'BitEthInitialCollateralSet',
```

Comment:
This seems really roundabout and error prone. Can we do something like: https://stackoverflow.com/questions/10314338/get-name-of-object-or-class

Import them all from the `./ts` folder and then use this?